### PR TITLE
Update a few things

### PR DESCRIPTION
**Pull request by JJL772**
Basically just pulling in changes from p2ce itself.

also made gameinfo a bit more readable.

### CODE REVIEW

File: cfg/mounts.kv [hunk 1]
```diff
@@ -13,81 +16,245 @@
...
+	//		"vpk" "lostcoast_pak"
+	//	}
+	// }
+	// **********************
+
+	// Team Fortress 2 (AppId 440)
+	// Known to override a few hl2 textures with tf2 counterparts, e.g.: hl2 shotgun
+	// worldmodel and concrete walls used across spytech environments
+	// **********************
+	 440
```
Comment (AWildErin) [inline cfg/mounts.kv:85]: This needs to be commented out.

File: cfg/mounts.kv [hunk 2]
```diff
@@ -13,81 +16,245 @@
...
+			"vpk" "tf2_sound_misc"
+			"vpk" "tf2_sound_vo_english"
+			"vpk" "tf2_textures"
+		}
+	 }
+	// **********************
+
+	// Half-Life 2 (AppID 220)
+	// **********************
+	 220
```
Comment (AWildErin) [inline cfg/mounts.kv:99]: Ditto

File: cfg/mounts.kv [hunk 3]
```diff
@@ -13,81 +16,245 @@
...
+	//	// not mounting ep1's folder + vpks will result in missing assets
+	// 	// persisting across the various ep2_outland maps.
+	//	//
+		// You do not need to mount AppID 380 (Half-Life 2: Episode One)
+		// if you have Ep2 installed, as both games share content between each other.
+		"episodic"
+		{
+			"vpk"	"ep1_pak"	
+		}
+	 }
```
Comment (AWildErin) [inline cfg/mounts.kv:55]: Ditto of below

File: cfg/mounts.kv [hunk 4]
```diff
@@ -13,81 +16,245 @@
...
-	//**********************
-
-	// HL2
-	// IF YOU WANT TO MOUNT HL2 CONTENT: UNCOMMENT (remove the double slashes //) OF THE LINES BETWEEN THE ASTERICKS. Do NOT remove the double slashes from the line with the astericks (*)
-	//**********************
-	//220
-	//{
-	//	"hl2"
+	// **********************
+
```
Comment (AWildErin) [inline cfg/mounts.kv:165]: nit: Weird empty line?

File: cfg/cvar_defaults.cfg [hunk 5]
```diff
@@ -1,25 +1,14 @@
...
-// This file defines default cvar values regardless
-// of the default value set in code.
-// Any cvar can be modified by this file, including
-// cheat vars regardless of the sv_cheats setting.
-// As such, this file should be included in a 
-// signed VPK so it cannot be modified by the 
-// end user. Otherwise, it'll be easy to cheat 
-// in your mod!
-// --------------------------------------------------
+// ---------------------------------------------
```
Comment (AWildErin) [inline cfg/cvar_defaults.cfg:1]: Do we really need to have this here? I don't think this comment needs to transfer over, and probably should retain the old comment above imho.

**Review by AWildErin — CHANGES_REQUESTED** [review]
The main thing that needs to change before this can be merged is the mounts.kv stuff. Other than that it's fine, the other comments are mainly nits

File: cfg/mounts.kv (hunk 1 above)
Comment (JJL772) [reply cfg/mounts.kv:85]: Yeah just realized I copied from my p2ce install

File: cfg/cvar_defaults.cfg [hunk 6]
```diff
@@ -33,6 +29,9 @@ m_customaccel_exponent 1.3
 mat_accelerate_adjust_exposure_down 3
 mat_vignette_enable 0
 
+// rich presence
+cl_richpresence_discord_client_id 747154846817452045
```
Comment (AWildErin) [inline cfg/cvar_defaults.cfg:33]: This is redundant, it's set later in the file.

**Review by AWildErin — APPROVED** [review]
One final thing! Other than this tiny change it looks good to me, so I'm gonna approve it and once that change is done it can be merged as it's just removal of a duplicated line.

**Pull request merged**